raiden/utils/cli.py

```diff
@@ -203,17 +203,17 @@ def command(name=None, cls=GroupableOptionCommand, **attrs):
 
 
 def group(name=None, **attrs):
-    return click.group(name, **{"cls": GroupableOptionCommandGroup, **attrs})
+    return click.group(name, **{"cls": GroupableOptionCommandGroup, **attrs})  # type: ignore
 
 
 def option(*args, **kwargs):
-    return click.option(*args, **{"cls": GroupableOption, **kwargs})
+    return click.option(*args, **{"cls": GroupableOption, **kwargs})  # type: ignore
 
 
-def option_group(name: str, *options: List[Callable]):
+def option_group(name: str, *options: Callable):
     def decorator(f):
         for option_ in reversed(options):
-            for closure_cell in option_.__closure__:
+            for closure_cell in option_.__closure__:  # type: ignore
```

Comment:
`Callable`  might be wrong, but I don't have a better idea, as `Option` isn't callable.

Reply:
`Option` isn't callable but `click.option` returns a decorator so this should be fine.